src/platyPS/platyPS.psm1

```diff
@@ -114,8 +114,10 @@ function New-MarkdownHelp
 
         [Parameter(ParameterSetName="FromMaml")]
         [string]
-        $ModuleGuid = "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
+        $ModuleGuid = "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX",
 
+        [switch]
+        $ExcludeDontShow
```

Comment:
Do we need a separate switch for it? Should it be the new default behavior? Perhaps the opposite switch would be preferable.

Reply:
We had a discussion about it with @joeyaiello and it made sense that not everyone would want this behavior as default. The specific use case for DontShow relates to dynamic parameters, where, you have certain parameters show up dynamically, so the static parameters don't need to be shown during tab completion. That's my understanding, at least.